omod/src/main/java/org/openmrs/module/adminui/page/controller/metadata/privileges/ManagePrivilegesPageController.java

```diff
@@ -22,5 +38,40 @@
 	public void get(PageModel model, @SpringBean("userService") UserService userService) {
 		model.addAttribute("privileges", userService.getAllPrivileges());
 	}
-	
+
+
+    public String post(PageModel model, 
+            		   @RequestParam(value = "privilegeName", required = false) @BindParams Privilege privilege,
+                       @RequestParam("action") String action,
```

Comment:
action is not needed since privileges are not retireable, so this method only gets called for purge